drivers/SmartThings/zwave-garage-door-opener/src/mimolite-garage-door/init.lua

```diff
@@ -61,12 +61,15 @@ end
 
 local function switch_binary_report_handler(self, device, cmd)
   local event = nil
-  if device:get_latest_state("main", capabilities.contactSensor.ID, capabilities.contactSensor.contact.NAME) == "closed" then
-    event = capabilities.doorControl.door.opening()
-  else
-    event = capabilities.doorControl.door.closing()
+  -- TODO: someone check my math against the DTH here:
+  if cmd.args.value == 0 then
+    if device:get_latest_state("main", capabilities.contactSensor.ID, capabilities.contactSensor.contact.NAME) == "closed" then
+      event = capabilities.doorControl.door.opening()
+    else
+      event = capabilities.doorControl.door.closing()
+    end
+    device:emit_event(event)
```

Comment:
@greens 
`else` block might be missing here in case of `cmd.args.value > 0` to emit `switch.ON` events: 

[DTH code for the reference](https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/28d1f9f5264c9ad112c686b08ecfecd39701eee6/devicetypes/smartthings/mimolite-garage-door-controller.src/mimolite-garage-door-controller.groovy#L124)

Reply:
the switch capability was dropped with the transition to drivers